src/Tribe/Views/V2/Hooks.php

```diff
@@ -55,14 +54,11 @@ protected function add_actions() {
 		add_action( 'rest_api_init', [ $this, 'register_rest_endpoints' ] );
 		add_action( 'tribe_common_loaded', [ $this, 'on_tribe_common_loaded' ], 1 );
 		add_action( 'wp_head', [ $this, 'on_wp_head' ], 1000 );
-
-		// Hide sensitive event info if post is password protected.
-		add_action( 'the_post', array( $this, 'action_manage_sensitive_info' ) );
-
 		add_action( 'tribe_events_pre_rewrite', [ $this, 'on_tribe_events_pre_rewrite' ] );
 		add_action( 'wp_enqueue_scripts', [ $this, 'action_disable_assets_v1' ], 0 );
 		add_action( 'tribe_events_pro_shortcode_tribe_events_after_assets', [ $this, 'action_disable_shortcode_assets_v1' ] );
 		add_action( 'updated_option', [ $this, 'action_save_wplang' ], 10, 3 );
+		add_action( 'the_post', [ $this, 'manage_sensitive_info' ] );
```

Comment:
This action was actually causing a problem when a password protected post appears in the loop. With your value filtering that you've added elsewhere, I suspected that this action is not needed for v2, so I grabbed this branch and removed this hook to see what happens. What results is precisely what we want: sanitized content _without_ impacting the presence of the **Get Tickets**/**RSVP Now** links. See here:

**List View**

![Screen Shot 2019-12-23 at 10 56 33 AM](https://user-images.githubusercontent.com/430385/71367536-5c67e180-2573-11ea-8b8c-bc9cca0827ea.png)

**Single Post**

![Screen Shot 2019-12-23 at 10 56 16 AM](https://user-images.githubusercontent.com/430385/71367537-5e31a500-2573-11ea-9b1c-d367a0a63531.png)

Reply:
Follow-up: I tested against the **Default Events Template** and the action is still necessary for `is_single()` pages.

Reply:
Putting this back into Progress to address this.